app/domain/errors.rb

```diff
@@ -370,27 +370,27 @@ module AuthnJwt
       InvalidIssuerConfiguration = ::Util::TrackableErrorClass.new(
         msg: "Issuer authenticator configuration is invalid. You should configured as authenticator variables: " \
               "'{0-resource-name}' or one of the following: '{1-resource-name}','{2-resource-name}'",
-        code: "CONJ00070E"
+        code: "CONJ00170E"
```

Comment:
i am guessing these are pushed by mistake?

Reply:
No its not
https://github.com/cyberark/conjur/pull/2148#discussion_r627614858